phpBB/styles/prosilver/theme/icons.css

```diff
@@ -85,3 +85,26 @@
     line-height: .9;
     height: 12px;
 }
+
+
+blockquote cite:before, .uncited:before {
+	-webkit-font-smoothing: antialiased;
+	-mos-osx-font-smoothing: grayscale;
+	font-family: FontAwesome;
+	font-size: 14px;
+	font-weight: normal;
+	font-style: normal;
+	font-variant: normal;
+	line-height: 1;
+	text-rendering: auto;
+	display: inline-block;
+	padding-right: 5px;
+	content: '\f10d'; /* Font Awesome quote-left */
+}
+
+.rtl blockquote cite:before, .rtl .uncited:before {
+	padding-right: 0;
+	padding-left: 5px;
+	content: '\f10e'; /* Font Awesome quote-right */
```

Comment:
I've taken a fresh look using phpBB 3.2.0 as a baseline and followed your point 1. In colours.css the `background-image` property points to a gif of size 15x12. The size of the image is implicit and not specified as a property. In content.css and bidi.css a margin of 20px is specified, creating an implicit padding of 5px. 

In making the change to use Font Awesome icons, we're making all these properties explicit. That's a good thing.

To make sure I fully understand what you're suggesting as changes, icons.css should only have the following added: 
```
blockquote cite:before, .uncited:before {
    content: '\f10d'; /* Font Awesome quote-left */
}

.rtl blockquote cite:before, .rtl .uncited:before {
    content: '\f10e'; /* Font Awesome quote-right */
}
```
and the rest of the properties should be specified in content.css and bidi.css. Is this correct?